samples/nrf9160/modem_shell/README.rst

```diff
@@ -36,6 +36,10 @@ LTE link control changes and queries the state of the LTE connection.
 Many of the changes are applied when going to online mode for the next time.
 You can store some link subcommand parameters into settings, which are persistent between sessions.
 
+Rel14 features are enabled by default, i.e., they are set when going into normal mode and this applies to bootup too.
```

Comment:
When mentioning rel14 for the first time, maybe use the full term "3GPP Release 14 features" to be clear. Also do you think we should list what are those features? Even if the reader was familiar with 3GPP releases, it is not evident what is included in the set. Or is it enough to list them in AT-cmd manual?

Reply:
You are right that we need something more about those features. However, I'm not keen on adding the feature lists to MOSH documentation. I'll try to get a reference to AT command manual.